### PR TITLE
SERVER-88684: Mixed10KThreads find operations limited to 10

### PR DESCRIPTION
**Pull request by miguelangelnieto**
The read queries match on an increasing set of documents over time because we insert more data that matches the query predicate over time. Read throughput is directly influenced by write throughput. To reduce that influence, the find query will only return 10 documents per query.

### CODE REVIEW

**Review by miguelangelnieto — COMMENTED** [review]
Patch test running here: https://spruce.mongodb.com/version/6618f6d5b363c7000724da3e/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC

**Review by thessem — APPROVED** [review]
We're fixing the code review required from us on docs updates, so leaving a tick to get out of the way.

File: src/workloads/scale/Mixed10KThreads.yml [hunk 1]
```diff
@@ -22,6 +23,7 @@ Keywords:
 dbname: &dbname mixed_10k
 DocumentCount: &NumDocs 10000
 CollectionCount: &NumColls 1
+Limit: &limit 100
```
Comment (jimoleary) [inline src/workloads/scale/Mixed10KThreads.yml:26]: This should be 10 and used on line 77 or deleted?

**Review by jimoleary — COMMENTED** [review]
Remove the limit anchor on line 20 ... it's not being used.

File: src/workloads/scale/Mixed10KThreads.yml (hunk 1 above)
Comment (miguelangelnieto) [reply src/workloads/scale/Mixed10KThreads.yml]: Removed, thanks.

**Review by jimoleary — APPROVED** [review]
LGTM 

We need to inform the Build Barons about the related jump in throughput.

**Pull request merged**